neo4j/templates/_helpers.tpl

```diff
@@ -96,7 +96,12 @@ If no password is set in `Values.neo4j.password` generates a new random password
     {{- end -}}
     {{- $ignored := set .Values.neo4j "password" $password }}
   {{- end -}}
+  {{- if not .Values.neo4j.passwordFromSecret | and .Values.neo4j.password  -}}
```

Comment:
i think its more readable if we write it this way, the pipes are quite confusing

and .(Values.neo4j.password) (not .Values.neo4j.passwordFromSecret)

Reply:
I can try that